Flow.Rmd

```diff
@@ -123,13 +169,407 @@ gating(gt, gs)
 
 Now that the workflow is finished, here is a look at all cell types identified:
 
-```{r plot-cell-types, fig.height=8, fig.width=8}
+```{r flow-plot-cell-types, fig.height=8, fig.width=8}
 autoplot(gs[[1]], strip.text = "gate", bins=100, merge=F, axis_inverse_trans=F) + flow_theme
 ```
 
 This should then be comparable to what was in the [OMIP-021 publication](https://onlinelibrary.wiley.com/doi/full/10.1002/cyto.a.22475), and here are relevant figures demonstrating the similarity of the cell subsets captured:
 
-```{r OMIP-021-figure, fig.align='center', fig.cap='Gating strategy illustrated with annotated figures from OMIP-021', echo=FALSE}
-knitr::include_graphics('images/OMIP021.png')
+```{r flow-OMIP-021-figure, results='asis', fig.align='center', fig.cap='Gating strategy illustrated with annotated figures from OMIP-021', echo=FALSE, out.width = '100%'}
```

Comment:
Was `results='asis'` the trick to getting this to display correctly?

Reply:
Yep, that seemed to do the trick.